examples/aggregator/gcp/pod.yaml

```diff
@@ -10,11 +10,26 @@ spec:
       image: gcr.io/oak-ci/oak-aggregator:latest
       ports:
         - containerPort: 8080
+      volumeMounts:
+        - name: tls-secret
+          mountPath: '/etc/oak-secrets'
+          readOnly: true
+      args:
+        [
+          '--application=config.bin',
```

Comment:
Configuration file may not always be called `config.bin`
E.x.:
https://github.com/project-oak/oak/blob/3619cab6406ed292960a1e5371d0d5b11a378909/examples/hello_world/config/BUILD#L26-L49

Reply:
I agree, but this `pod.yaml` file is for this specific example where it will be called that. With the decision to move the arguments to `pod.taml` this must be specified in a file that is currently not autogenerated. Is there a bazel build rule to build pod files like these?

Reply:
No, currently there is only a rule to create Docker images